content/knowledge-base/install-unity-version.md

```diff
@@ -0,0 +1,117 @@
+---
+description: How to install a different Unity
+title: Installing a different Unity version
+weight: 13
+---
+
+# Installing different Unity versions
+Each build machine image has a specific version of Unity installed. You can find out the specific Unity version by consulting the build machine specification in the documentation. For example, the [macOS build machine specification (Xcode 13.3+ / Unity)](https://docs.codemagic.io/specs/versions-macos-xcode-13-3/#unity-2020331f1) has Unity version version 2020.3.31f1.
```

Comment:
I wouldn't mention any specific Unity versions as these are likely to change sooner or later and we'll probably forget to update this doc here. Maybe there's some other way you can link to the build machine docs.

Reply:
Perhaps link to the most recent image per platform like this: Each build machine image for macOS, Windows, or Linux has a specific version of Unity installed.

Reply:
Will remove the link to specific versions.